test/integration/swap/common.js

```diff
@@ -132,6 +132,12 @@ async function refund (chain, initiationTxId, secretHash, swapParams) {
   return refundTxId
 }
 
+async function waitRefund(chain, initiationTxId, secretHash, swapParams) {
```

Comment:
Can the `refund` function use the same model as `claimAndVerify`?

Reply:
@monokh You mean passing the secret instead of secret hash bot for waitRefund() and refund() ?

Reply:
I mean `refund` can be `refundAndVerify` which both submits the refund tx and tries to retrieve it using the find function. This way anywhere we do refund we are also testing that the find function works too